convert2rhel/pkghandler.py

```diff
@@ -28,6 +28,8 @@
 # Limit the number of loops over yum command calls for the case there was
 # an error.
 MAX_YUM_CMD_CALLS = 2
+_VERSIONLOCK_FILE_PATH = '/etc/yum/pluginconf.d/versionlock.list'
+VERSIONLOCK_FILE = utils.RestorableFile(_VERSIONLOCK_FILE_PATH)
```

Comment:
```suggestion
versionlock_file = utils.RestorableFile(_VERSIONLOCK_FILE_PATH)  # pylint: disable=C0103
```
To be consistent in the project, use lowercase as is the case of all the other RestorableFile instances (e.g. `system_release_file` or `yum_conf`).